src/components/StripePaymentAddress/StripePaymentAddress.css

```diff
@@ -0,0 +1,30 @@
+@import '../../marketplace.css';
+
+.root {
+}
+
+.paymentAddressField {
+  padding-top: 38px;
+}
+
+.formRow {
```

Comment:
I'd recommend that you try to keep these rules in some kind of order.

For example, _flex-shrink_ is something that expects parent component to use flexbox layout. => it would be better to have those rules first.
https://github.com/sharetribe/flex-template-web/blob/c30a6ab6e22e1bd6f4c155d435a0ea485b76c33c/src/components/LayoutSideNavigation/LayoutSideNavigation.css#L11-L18

Btw. `.sectionContainer` is used in the code, but there's no rule for it. (maybe it should refer to `.root`?)